Common/Data/Model/File.swift

```diff
@@ -14,33 +14,37 @@ public final class File: NSManagedObject {
     }
 
     @NSManaged public var id: String
-    @NSManaged public var cacheURL_: String? // swiftlint:disable:this identifier_name
+    @NSManaged public var remoteURL_: String? // swiftlint:disable:this identifier_name
```

Comment:
I prefer not using an underscore at the end of the variable name (like swiftlint tries to enforce). You know you can declared attributes of `NSManagedObjects` as private.